helm/templates/env.yaml

```
@@ -9,25 +9,19 @@ data:
   CONNAISSEUR_NAMESPACE: {{ .Release.Namespace }}
```

i have a vision:

```suggestion
  CONNAISSEUR_NAMESPACE: {{ .Release.Namespace }}
  KUBE_API_TOKEN_PATH: /var/run/secrets/kubernetes.io/serviceaccount/token
  KUBE_API_CA_PATH: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
  LOG_LEVEL: {{.Values.application.logLevel | default "INFO"}}
  {{- with .Values.application.features }}
  DETECTION_MODE: {{ default false .detectionMode | quote }}
  AUTOMATIC_CHILD_APPROVAL_ENABLED: {{ default true .automaticChildApproval | quote }}
  AUTOMATIC_UNCHANGED_APPROVAL: {{ default true .automaticUnchangedApproval | quote }}
  {{- end }}
  {{- if .Values.alerting }}
  CLUSTER_NAME: {{ default "not specified" .Values.alerting.clusterIdentifier }}
  {{- end }}
  KUBE_VERSION: {{ .Capabilities.KubeVersion }}
```

then have a new function in `utils.py`, something like `feature_flag_on(key: str)` which can parse the string bool values into actual bools. also put all the feature flag keys into `constants.py`.

or keep things as they are and i'll make a seperate PR with these changes 🤷

---

Since that's no longer refactoring of the helm/values.yaml and this MR is already quite big, I'd say it'd be better to move it to a dedicated MR instead

---

i will take care of this.